modules/transformer-kinesis/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/AppId.scala

```diff
@@ -0,0 +1,7 @@
+package com.snowplowanalytics.snowplow.rdbloader.transformer
+
+object AppId {
+  val appId: String = java.util.UUID.randomUUID.toString
+
+  System.out.print(appId)
```

Comment:
I would prefer if the `Main` class logged the app id using a Slf4jLogger.  Otherwise we get a cryptic uuid in the logs with no context about what it means.